Packs/Mattermost/Scripts/MattermostAskUser/MattermostAskUser.yml

```
@@ -32,11 +32,17 @@ args:
   - "false"
   description: Indicates whether to use one-time entitlement or a persistent one.
   defaultValue: "false"
+- defaultValue: Thank you **{user}**. You have answered **{response}** .
+  description: The reply to send to the user. Use the templates {user} and {response} to incorporate these in the reply. (i.e., "Thank you **{user}**. You have answered **{response}**.").
+  name: reply
+- defaultValue: 1 day
+  description: Time until the question expires. For example - 1 day. When it expires, a default response is sent. Default value is 1 day.
```

I think that the str "Default value is 1 day. is added automatically no"?
```suggestion
  description: Time until the question expires. For example - 1 day. When it expires, a default response defined under the 'defaultResponse' argument is sent. 
```

---

Yes